source/js/schemes/muse.js

```diff
@@ -157,7 +144,7 @@ $(document).ready(function() {
 
       sidebarToggleLines.close();
 
-      this.sidebarEl.velocity('stop').velocity({
+      this.sidebarEl.stop().velocity({
```

Comment:
L147 L153 L159, I'm not sure if velocity ≈ jQuery.animate(), but I replace them with animate. seems good...